docs/workforce-management-framework/upstream-1/master.adoc

```diff
@@ -7,19 +7,32 @@ include::topics/shared/attributes.adoc[]
 
 image::https://raw.githubusercontent.com/feedhenry-raincatcher/raincatcher-docs/master/docs/shared/images/logo-blue.png[logo]
 
```

Comment:
Not sure why this info is being added back for Logger  - there could be a reason unknown to me?